tests/test_util.py

```
@@ -6,66 +6,70 @@
 import pandas as pd
 import numpy as np
 
-import sys; sys.path.append("../")
-import missingno as msno
+import sys
+
+sys.path.append("../")
+from missingno.utils import nullity_filter, nullity_sort
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)